.github/workflows/build.yml

```diff
@@ -7,20 +7,23 @@ on:
     - '*'
     tags:
     - '*'
+  pull_request:
+    branches:
+    - '*'
 
 defaults:
   run:
     shell: bash
 
 jobs:
   build:
-    name: Build
+    name: Build python package
```

Comment:
This commit also does:

* fix typos
* bump upload and checkout actions
* add trigger on pull-request
* expand 'build release package'

aka there is a lot going on there, perhaps just state that in the commit-message

Reply:
I have expanded the git message